### PR TITLE
Update to elm version 0.19  

### PR DESCRIPTION
**Pull request by monperrus**
Hi Duane,

elm 0.19  introduces many non backward compatible changes, I followed the upgrade instructions at https://github.com/elm/compiler/blob/master/docs/upgrade-instructions/0.19.0.md

Hope you'll find this useful!

### CODE REVIEW

File: src/Hccb/Rect.elm [hunk 1]
```diff
@@ -13,9 +13,9 @@ rect x y w h =
     Rect (Point x y) (Dim w h)
 
 
-fromRect : Rect -> ( Float, Float, Float, Float )
-fromRect (Rect (Point x y) (Dim w h)) =
-    ( x, y, w, h )
+-- fromRect : Rect -> ( Float, Float, Float, Float )
+-- fromRect (Rect (Point x y) (Dim w h)) =
+--     ( x, y, w, h )
```
Comment (canadaduane) [inline src/Hccb/Rect.elm:18]: Out of curiosity, why was it necessary to comment fromRect out?

File: README.md [hunk 2]
```diff
@@ -25,13 +25,17 @@ I discovered HCCB, among other formats, through [Specifications For Popular 2D B
...
 ## Getting started
 
-You need to have [Elm](http://elm-lang.org/) 0.18 installed on your machine.
+Install [Elm](https://github.com/elm/compiler/blob/master/installers/linux/README.md) in your $PATH.
 
-Compile the samples in this project with:
+Compile Hccb
 
-    cd examples; make
+    elm make elm make src/Hccb.elm
```
Comment (canadaduane) [inline README.md]: It looks like there is one too many "elm make"s here?

File: README.md [hunk 3]
```diff
@@ -25,13 +25,17 @@ I discovered HCCB, among other formats, through [Specifications For Popular 2D B
...
 
-Then view it:
+Compile an example
+
+    elm make examples/Simple/Simple.elm
+
+Then view the example in <http://localhost:8000/examples/Simple/index.html> with:
```
Comment (canadaduane) [inline README.md]: I don't think the angle brackets are necessary for markdown to know this is a link

**Review by canadaduane — CHANGES_REQUESTED** [review]
Thanks for the update! I've added a few comments, should be small changes, and then I will merge it in.

File: src/Hccb/Rect.elm (hunk 1 above)
Comment (monperrus) [reply src/Hccb/Rect.elm:18]: Because elm 0.19 forbids tuples of more than 3 elements :-(

File: README.md (hunk 2 above)
Comment (monperrus) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (monperrus) [reply README.md]: done

**Comment by monperrus** [comment]
Pushed again!

**Comment by canadaduane** [comment]
Thanks! Looks great.

**Comment by monperrus** [comment]
Thanks! It was fun to learn a little bit about elm.

**Pull request merged**